### PR TITLE
Add origin trial details to SIMD feature guide

### PR DESCRIPTION
**Pull request by dtig**

### CODE REVIEW

File: src/features/simd.md [hunk 1]
```diff
@@ -142,8 +142,14 @@ Another interesting set of demos that makes use of SIMD for smooth experience, c
...
+
+Ongoing experimental support is available on a recent Chrome Canary as detailed [above](#using-webassembly-simd), with the use of latest-upstream Emscripten toolchain.
+ 
 ## Future work
 
 The current SIMD proposal is in Phase 2, so the future work here is to push the proposal forward in the standardization process. Fixed width SIMD gives significant performance gains over scalar, but it doesn’t effectively leverage wider width vector operations that are available in modern hardware. As the current proposal moves forward, some future facing work here is to determine the feasibility of extending the proposal with longer width operations.
-
-To try out current experimental support, use the latest-upstream Emscripten toolchain, and a recent Chrome Canary as detailed [above](#using-webassembly-simd). Please note that as the support is experimental, we are actively working on feature completion and performance so some breakage, or performance inconsistencies are possible.
```
Comment (RReverser) [inline src/features/simd.md:149]: Do we not want people to use this flag for experimentation anymore?

Comment (dtig) [reply src/features/simd.md:149]: I've moved an abridged version of this text to be with the origin trial text, see the last sentence of that section.

Comment (RReverser) [reply src/features/simd.md:149]: Ah cool, thanks.

**Comment by RReverser** [comment]
Looks like linter fails on CI due to some trailing spaces.

**Comment by dtig** [comment]
> Looks like linter fails on CI due to some trailing spaces.

Fixed now.

**Pull request merged**